### PR TITLE
Organize cleanup recipes

### PR DESCRIPTION
**Pull request by pway99**

### CODE REVIEW

File: rewrite-java/src/main/resources/META-INF/rewrite/static-analysis-cleanup.yml [hunk 1]
```diff
@@ -20,53 +20,64 @@ displayName: Common static analysis issues
 description: Resolve common static analysis issues discovered through 3rd party tools
```
Comment (tkvangorder) [inline rewrite-java/src/main/resources/META-INF/rewrite/static-analysis-cleanup.yml:20]: can you go through each of the commented-out recipes and make a determination if these should just always be part of the auto-remediation of static analysis issues?  

The only one that I know that should NOT be on this list is the `FixSerializableFields`, this really needs some human intervention. 

If you have questions on the others, just do a git blame and ask the original author if it was just an oversight for it not being included.

**Review by tkvangorder — CHANGES_REQUESTED** [review]
This looks like a reasonable reorg: 

One recipe to remediate checkstyle.
One recipe to fix common code cleanup
One for sonar issues

File: rewrite-java/src/main/resources/META-INF/rewrite/static-analysis-cleanup.yml (hunk 1 above)
Comment (pway99) [reply rewrite-java/src/main/resources/META-INF/rewrite/static-analysis-cleanup.yml:20]: sure.  thanks @tkvangorder

**Comment by tkvangorder** [comment]
@pway99 There are still a few commented-out recipes in the list, once these are either removed/added, I am good to merge this.

**Pull request merged**